package.json

```diff
@@ -12,7 +12,8 @@
     "README.md",
     "dist/*.js",
     "src/**",
-    "index.mjs"
+    "index.mjs",
+    "flow-typed/*.js"
```

Comment:
Why is this exported?

Reply:
agreed with @Rob--W to don't export the custom flow definitions for the third party dependencies used internally for now, we can include them if any other package depends from web-ext and would need them.